lua/packer.lua

```diff
@@ -49,6 +49,7 @@ local config_defaults = {
   },
   display = {
     non_interactive = false,
+    oneline_results = true,
```

Comment:
Not sure that this is the best option name, it seems very specific to the current UI implementation, but also doesn't exactly clarify what exactly might be expected to be 1 line there are lots of results in packer. I think something along the lines of compact or default_closed etc. it might be one day that we can't guarantee 1 line, and it just becomes a matter of trying to make things more concise than full output

Reply:
Good point! Changed to `compact`